src/main/java/com/connexta/transformation/impl/ServiceRegistryConsumerImpl.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformation.impl;
+
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import com.connexta.transformation.api.ServiceRegistryConsumer;
+import com.connexta.transformation.api.ServiceRegistryProducer;
+
+@Component
+public class ServiceRegistryConsumerImpl implements ServiceRegistryConsumer {
+  private final Logger LOGGER = LoggerFactory.getLogger(ServiceRegistryConsumerImpl.class);
+
+  public ServiceRegistryConsumerImpl() {
```

Comment:
@mdang8 - ✏️ Do we need this constructor for something in the future?

Reply:
I changed it to inject the AMQP template in the constructor.